tests/features/collection/propose.feature

```diff
@@ -135,3 +135,27 @@ Feature: Proposing a collection
     When I click on element ".js-form-item-field-ar-closed-value label"
     Then the option "Only members can create new content." should be selected
     And the option "Only collection facilitators can create new content." should not be selected
+
+  @javascript
+  Scenario: Propose collection form fields should be organized in groups.
```

Comment:
There are no failures in the [CPHP build](https://continuousphp.com/git-hub/ec-europa/joinup-dev/build/77d4057a-9786-4c6c-b961-655b72d62ff8/output). CPHP test results are the only one that we take as reference. Also, it doesn't fail locally for me. Maybe you have some local instance issues?